Description: providing the Maven coordinates of the Debian-packaged javacc
 I don't understand why, but the standard debian/maven.rules mechanism does not
 allow one to recover the good coordinates.
Author: Pierre Gruet <pgt@debian.org>
Forwarded: not-needed
Last-Update: 2024-04-24

--- a/javaparser-core/pom.xml
+++ b/javaparser-core/pom.xml
@@ -31,8 +31,8 @@
     <build>
         <plugins>
             <plugin>
-                <groupId>com.helger.maven</groupId>
-                <artifactId>ph-javacc-maven-plugin</artifactId>
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>javacc-maven-plugin</artifactId>
                 <executions>
                     <execution>
                         <id>javacc</id>
--- a/pom.xml
+++ b/pom.xml
@@ -195,9 +195,9 @@
         <pluginManagement>
             <plugins>
                 <plugin>
-                    <groupId>com.helger.maven</groupId>
-                    <artifactId>ph-javacc-maven-plugin</artifactId>
-                    <version>4.1.5</version>
+                    <groupId>org.codehaus.mojo</groupId>
+                    <artifactId>javacc-maven-plugin</artifactId>
+                    <version>2.6</version>
                 </plugin>
                 <plugin>
                     <groupId>org.apache.maven.plugins</groupId>
